docs/source/release_notes/release-notes.rst

```diff
@@ -43,6 +43,22 @@ develop
 
 .. replace this with the release date and the above with v<tag> (the v makes the permalinks nice)
 
+.. list-table::
+    :widths: 5 40
+    :header-rows: 1
+
+    *    - Type
+         - Change
+
+    *    -
+         -
+
+=======
+v0.93.0
+=======
+
+25 June 2018
+
 .. list-table::
     :widths: 5 40
     :header-rows: 1
```

Comment:
nit: only thing here is maybe saying how the logging was erroneous before #3314 (i.e. it logged whether the host was blacklisted or not)

Reply:
agreed 👍